plugins/main/public/components/overview/vulnerabilities/dashboards/inventory/inventory.scss

```diff
@@ -1,14 +1,17 @@
 .vulsInventoryContainer {
-    height: calc(100vh - 104px);
-}
-
+  height: calc(100vh - 104px);
 
-.headerIsExpanded .vulsInventoryContainer {
-    height: calc(100vh - 153px);
-}
+  .euiDataGrid__virtualized {
```

Comment:
I suggest adding a comment to make clear why this style is necessary.

Reply:
done